### PR TITLE
vulnerability fixed, swagger reference removed

### PR DESCRIPTION
**Pull request by Monam-Bharti**
Issue: https://github.com/IBM/kubernetes-driver/issues/84
 List of related PRs. : None
 Project builds without any errors.
 Unit Tests updated (or created where needed) and all pass.
 You have ran manual functional “smoke” test (does product as a whole still work?).
 User Story meets the acceptance criteria - and passes. acceptance criteria should be understood at outset.
 There is a PR for update of internal documentation for the affected Project(s) - README etc.
 Description of the changes: https://github.com/IBM/kubernetes-driver/issues/84

### CODE REVIEW

**Review by owen-lynch-ibm — APPROVED** [review]

File: docs/install.md [hunk 1]
```diff
@@ -58,12 +58,6 @@ helm install kubedriver-<version>.tgz --name kubedriver -f custom-values.yml
 
 The driver runs with SSL enabled by default. The installation will generate a self-signed certificate and key by default, adding them to the Kubernetes secret "kubedriver-tls". To use a custom certificate and key in your own secret, override the properties under "apps.config.security.ssl.secret".
 
-### Confirm 
-
-You can confirm the driver is working by accessing the Swagger UI included to render the API definitions.
-
-Access the UI at `https://your_host:31684/api/driver/ui` e.g. [`http://localhost:31684/api/driver/ui`](http://localhost:31684/api/driver/ui)
-
```
Comment (hhellyer) [inline docs/install.md:66]: Does the user need another way to confirm the driver is working now that the swagger ui has been removed?

**Review by hhellyer — CHANGES_REQUESTED** [review]
I think we need to verify if the image size has been increased and probably clean up the cache.
It would be ideal if there was also another way for the user to confirm the driver was working properly now the swagger ui has been removed but there may not be an easy answer there.

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 FROM python:3.9-alpine
+RUN apk update && apk upgrade
```
[the remark this answers is not in the document]
Comment (Monam-Bharti) [reply docker/Dockerfile]: @hhellyer Made the required changes as suggested. 
Please review the changes and let us know if any other change is required.

**Review by hhellyer — APPROVED** [review]
I'm happy with the changes - @dvaccarosenna updated ignition in the `kubedriver/pkg_info.json` file this morning. That conflict should be resolved and the changes rebuilt and verified with the new level of ignition before this is merged.

**Review by owen-lynch-ibm — DISMISSED** [review]

**Comment by Monam-Bharti** [comment]
@hhellyer @owen-lynch-ibm 
Also, made required changes for IPv6 support in docker/Dockerfile.

**Review by owen-lynch-ibm — DISMISSED** [review]

**Review by hhellyer — DISMISSED** [review]

File: docs/install.md (hunk 1 above)
Comment (Monam-Bharti) [reply docs/install.md:66]: @hhellyer Only through `oc get pod kubedrivexxxxx` will show the status of the pod and no other endpoints are available to verify the same.

Comment (dvaccarosenna) [reply docs/install.md:66]: This driver uses the Ignition framework which, after the removal of the Swagger UI, has suggested using the `/management/health` endpoint instead: https://github.com/IBM/ignition/blob/develop/ignition/templates/driver_template/docs/index.md#install

So all drivers should be updated to suggest the same.

Comment (Monam-Bharti) [reply docs/install.md:66]: @dvaccarosenna @hhellyer 
Added ```http://<kubernetes-node-ip>:{(helm.node_port)}/management/health```

Comment (dvaccarosenna) [reply docs/install.md:66]: `{(helm.node_port)}` can be replaced with `31684` as we know that's the default node port

**Comment by Monam-Bharti** [comment]
@dvaccarosenna @hhellyer Can you please review the changes one more time and let me know if everything looks good.

**Review by owen-lynch-ibm — APPROVED** [review]

**Pull request merged**